cmd/lakectl/cmd/commit.go

```diff
@@ -9,31 +9,43 @@ import (
 	"github.com/treeverse/lakefs/pkg/uri"
 )
 
-var commitCreateTemplate = `Commit for branch "{{.Branch.Ref}}" completed.
+var (
+	errInvalidKeyValueFormat    = fmt.Errorf("invalid key/value pair - should be separated by \"=\"")
+	errEmptyMessage             = fmt.Errorf("commit with an empty message without specifying the \"--allow-empty-message\" flag")
+	allowEmptyCommitMessageFlag bool
+	commitCreateTemplate        = `Commit for branch "{{.Branch.Ref}}" completed.
 
 ID: {{.Commit.Id|yellow}}
 Message: {{.Commit.Message}}
 Timestamp: {{.Commit.CreationDate|date}}
 Parents: {{.Commit.Parents|join ", "}}
 
 `
+)
 
-var errInvalidKeyValueFormat = fmt.Errorf("invalid key/value pair - should be separated by \"=\"")
+const (
+	MessageFlagName           string = "message"
+	AllowEmptyMessageFlagName string = "allow-empty-message"
+	MetaFlagName              string = "meta"
```

Comment:
`string` is redundant - prefer to use without so it can be used for all `string` kind as it takes the type when it is used.